app/lib/server/startup/settings.js

```diff
@@ -1071,6 +1071,28 @@ settings.addGroup('General', function() {
 			type: 'boolean',
 		});
 	});
+	this.section('Timezone', function() {
+		this.add('Default_Timezone_For_Reporting', 'server', {
+			type: 'select',
+			values: [{
+				key: 'server',
+				i18nLabel: 'Default_Server_Timezone',
+			}, {
+				key: 'custom',
+				i18nLabel: 'Default_Custom_Timezone',
+			}, {
+				key: 'user',
+				i18nLabel: 'Default_User_Timezone',
+			}],
+		});
+		this.add('Default_Custom_Timezone', '', {
+			type: 'timezone',
+			enableQuery: {
+				_id: 'Default_Timezone_For_Reporting',
+				value: 'custom',
+			},
+		});
```

Comment:
The setting translations are missing(EN at least)
![Screen Shot 2021-08-10 at 1 34 04 PM](https://user-images.githubusercontent.com/59577424/128898195-c272c694-734e-4688-8ca0-4b8ba1b0a7f9.png)

Reply:
I added translations for english & spanish